knowledge_base/python_decorator/README.md

```diff
@@ -0,0 +1,78 @@
+# Python decorator
+
+This example shows how to package a **Python decorator–based job** so you can
+call regular Python functions from a Databricks job with minimal boilerplate.
+
+## How it works
+
+| File                                   | Purpose                                                        |
+|----------------------------------------|----------------------------------------------------------------|
+| `src/my_project/tasks.py`              | Defines decorated task functions.                              |
+| `src/my_project/jobs.py`               | Wires those tasks into a `Job` definition.                     |
+| `src/my_project/python_wheel_task.py`  | Implements `@python_wheel_task` decorator.                     |
+| `src/my_project/resources.py`          | Defines `load_resources` function to load `Job` definitions.   |
+
+### Creating a task function
+
+```python
+# src/my_project/tasks.py
+from my_project.python_wheel_task import python_wheel_task
+
+
+@python_wheel_task
```

Comment:
For this mini-framework, should this not just be called `@task`? In a more extensive version, we could always decide to use subtypes like `@task.sql` or move to top-level task types like `@sql_file_task`.